Game1.cs

```diff
@@ -85,16 +77,12 @@ protected override void Draw(GameTime gameTime)
         
         _spriteBatch.Begin(_camera, samplerState: SamplerState.PointClamp);
         
-            _escape.Draw(_spriteBatch);
-            _enemy.Draw(_spriteBatch);
-
-            CombatManager.Draw(_spriteBatch);
-            ItemManager.Draw(_spriteBatch);
-            
+        _escape.Draw(_spriteBatch);
+        CombatManager.Draw(_spriteBatch);
         _spriteBatch.End();
         
-        _spriteBatch.Begin(samplerState: SamplerState.PointClamp);
-            UiManager.Draw(_spriteBatch);
+        _spriteBatch.Begin();
```

Comment:
Hier wurde die Änderung vom dev wohl auch irgendwie nicht übernommen, deshalb ist das gui etwas verschwommen. Ich würde noch das SamplerState.PointClamp in spritebatch.Begin() einfügen :) Dann sollte es wieder auf dem richtigen Stand sein